marklogic-client-api/src/main/java/com/marklogic/client/dataservices/impl/OutputEndpointImpl.java

```diff
@@ -59,7 +62,59 @@ private BulkOutputCallerImpl(OutputEndpointImpl endpoint) {
         @Override
         public void awaitCompletion() {
```

Comment:
Would it be appropriate to add a guard to make sure the output consumer is not null?